yt/go/yt/internal/encoder.go

```diff
@@ -244,6 +246,64 @@ func (e *Encoder) RemoveMember(
 	return
 }
 
+func (e *Encoder) SetUserPassword(
+	ctx context.Context,
+	user string,
+	newPassword string,
+	currentPassword string,
+	options *yt.SetUserPasswordOptions,
+) (err error) {
+	newPasswordSHA256 := encodeSHA256(newPassword)
+	currentPasswordSHA256 := ""
+	if currentPassword != "" {
+		currentPasswordSHA256 = encodeSHA256(currentPassword)
+	}
+
+	call := e.newCall(NewSetUserPasswordParams(user, newPasswordSHA256, currentPasswordSHA256, options))
+	err = e.do(ctx, call, func(res *CallResult) error {
+		return nil
+	})
+	return
+}
+
+func (e *Encoder) IssueToken(
+	ctx context.Context,
+	user string,
+	password string,
+	options *yt.IssueTokenOptions,
+) (token string, err error) {
+	passwordSHA256 := ""
+	if password != "" {
```

Comment:
Don't you want to put this if inside `encodeSHA256` func?

Reply:
Actually I don't think it would make code better, because 1) `encodeSHA256` just encodes some string and doesn't know is it password or empty string or something 2) `encodeSHA256` is used also for tokens in `RevokeToken` and shouldn't be empty 3) checking password/token emptiness here is an area of responsibility of api method business logic, not helper, i think.